MapData.py

```diff
@@ -0,0 +1,41 @@
+import os
+import threading
+from enum import Enum
+
+import matplotlib.pyplot as plt
+from scipy.misc import imresize
+
+from detail import LOCAL
+
+class MapDataFieldNamesEnum(Enum):
+    ZOOM = 'zoom'
+    RADIUS = 'radius'
+    IMAGE = 'image'
+    MARKER_PATH = 'marker path'
+    MARKER = 'marker'
+    ANNO_BOX = 'annotation box'
+    LOCATION = 'location'
+
+class MapData:
+    def __init__(self):
+        self.lock = threading.Lock()
+
+        # Map UI attributes
+        self.mapData = {}
+        self.mapData[MapDataFieldNamesEnum.ZOOM.value] = 20
+        self.mapData[MapDataFieldNamesEnum.RADIUS.value] = 0.1
```

Comment:
These are just constants which are used only in MappingThread I believe. I think it would be better to just have them as constants in that file.